cmd/nf-dump-pcap/main.go

```diff
@@ -15,14 +15,14 @@ import (
 
 	"github.com/google/gopacket"
```

Comment:
Don't we also want to pin all the transitive dependencies? I think traditionally we've cloned everything.